src/lib/Form/EventListener/AddSiteAccessFieldBasedOnContentListener.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Form\EventListener;
+
+use eZ\Publish\API\Repository\Values\Content\Location;
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+use EzSystems\EzPlatformAdminUi\Form\Data\Content\CustomUrl\CustomUrlAddData;
+use EzSystems\EzPlatformAdminUi\Siteaccess\NonAdminSiteaccessResolver;
+use Symfony\Component\Form\ChoiceList\Loader\CallbackChoiceLoader;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormEvent;
+
+class AddSiteAccessFieldBasedOnContentListener
+{
+    /** @var \EzSystems\EzPlatformAdminUi\Siteaccess\NonAdminSiteaccessResolver */
+    private $nonAdminSiteaccessResolver;
+
+    /** @var \eZ\Publish\Core\MVC\ConfigResolverInterface */
+    private $configResolver;
+
+    /**
+     * @param \EzSystems\EzPlatformAdminUi\Siteaccess\NonAdminSiteaccessResolver $nonAdminSiteaccessResolver
+     * @param \eZ\Publish\Core\MVC\ConfigResolverInterface $configResolver
+     */
```

Comment:
Redundant
```suggestion
```

Reply:
Why is it redundant? We tend to provide full namespaces in almost all constructors with DI.

Reply:
> Why is it redundant? We tend to provide full namespaces in almost all constructors with DI.

CS on that changed a year ago or so. We agreed that if the PHPDoc does not provide any extra information, it's redundant. Constructor has strict types and there's no extra description in your PHPDoc. Documenting systems should be able to display correct information whether this PHPDoc is present or not. 

On the other hand, if some arguments were not strictly typed or typed as a generic `array`, then PHPDoc would be justified. Also some verbose explanation of the meaning of a param justifies placing an extra doc.